include/functions.h

```diff
@@ -1564,9 +1564,9 @@ void AudioMgr_HandlePreNMI(AudioMgr* audioMgr);
 void AudioMgr_ThreadEntry(void* arg0);
 void AudioMgr_Unlock(AudioMgr* audioMgr);
 void AudioMgr_Init(AudioMgr* audioMgr, void* stack, OSPri pri, OSId id, Scheduler* sched, IrqMgr* irqMgr);
-void TitleSetup_InitImpl(GameState* gameState);
-void TitleSetup_Destroy(GameState* gameState);
-void TitleSetup_Init(GameState* gameState);
+void Setup_InitImpl(GameState* this);
+void Setup_Destroy(GameState* this);
+void Setup_Init(GameState* this);
```

Comment:
id use `thisx`
even tho it doesnt have an instance itself, it theoretically can (and probably did?)

and maybe we can drop this down near the other gamestates just to keep them near each other. not that it matters too much i guess

Reply:
infact, I think itd be a good idea to make a SampleState struct that just wraps GameState and is used in this file

Reply:
oh and also, InitImpl is internal and can be removed